runtime/new-recipe.js

```diff
@@ -219,10 +219,8 @@ class ViewConnection extends Connection {
   toString(nameMap) {
     let result = [];
     result.push(this.name || '*');
-    if (['in', 'out', 'inout'].indexOf(this.direction) == -1)
-      result.push(`{${this.direction}}`)
-    else
-      result.push({'in': '<-', 'out': '->', 'inout': '='}[this.direction]);
+    // TODO: better deal with unspecified direction.
+    result.push({'in': '<-', 'out': '->', 'inout': '='}[this.direction] || '=');
```

Comment:
could we "|| this.direction" for now at least so it's clear what the incorrect values being specified are?